apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -74,12 +74,17 @@ export default class LibraryManagerDialog extends React.Component {
     this.setState({importLibraryId: event.target.value});
   };
 
-  addLibrary = channelId => {
-    let libraryClient = new LibraryClientApi(channelId);
+  importLibrary = (channelId, versionId) => {
     // TODO: Check for naming collisions between libraries.
-    libraryClient.getLatest(
+    let libraryClient = new LibraryClientApi(channelId);
+    libraryClient.getVersion(
```

Comment:
TODO: update fetchLatest controller to return the version number alongside the library
https://codedotorg.atlassian.net/browse/STAR-805